### PR TITLE
chore(kinesisfirehose-alpha): add ca-west-1 Firehose CIDR block

### LINKED ISSUE #29874: aws-kinesisfirehose-alpha: Needs support for `ca-west-1`
### Describe the bug

Creating a DeliveryStream in `ca-west-1` fails at synth time with `Error: Mapping doesn't contain top-level key ca-west-1`

### Expected Behavior

Configured constructs in a stack set to deploy to `ca-west-1` using `aws-cdk-lib` 2.134 and `@aws-cdk/aws-kinesisfirehose-alpha` 2.134-alpha. Expected a clean compile.

### Current Behavior

```
Error: Mapping doesn't contain top-level key 'ca-west-1'
...
```

when attempting to compile.

### Reproduction Steps

Just take any normal code example of `DeliveryStream` like your unit tests and synth it in `ca-west-1`.

### Possible Solution

`aws-kinesisfirehose-alpha` uses a mapping of CIDR blocks (https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts#L768) per region. This list is incomplete (it does not include `ca-west-1`).

### Additional Information/Context

Basically identical issue to https://github.com/aws/aws-cdk/issues/27518, but for another region and the (now) newest CDK

### CDK CLI Version

2.134 

### Framework Version

_No response_

### Node.js Version

v18.18.0

### OS

Amazon Linux 2

### Language

TypeScript

### Language Version

TypeScript (4.9.4)

### Other information

_No response_

### PR DESCRIPTION
**Pull request by paulhcsun**
### Issue # (if applicable)

Closes #29874

### Description of changes

Add missing CIDR block for `ca-west-1` + update snapshots. 

CIDRs pulled from [here](https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html#using-iam-rs).

### Description of how you validated changes

integ tests

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json [hunk 1]
```diff
@@ -252,6 +252,9 @@
    "ca-central-1": {
     "FirehoseCidrBlock": "35.183.92.128/27"
    },
+   "ca-west-1": {
+    "FirehoseCidrBlock": "40.176.98.192/27/27"
```
Comment (jfuss) [inline packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json:256]: Should be `40.176.98.192/27`. There is an extra `/27` at the end.

**Review by jfuss — DISMISSED** [review]

File: packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json (hunk 1 above)
Comment (paulhcsun) [reply packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json]: Nice catch

Comment (paulhcsun) [reply packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json:256]: Not sure why but this keeps getting automatically updated with the extra `/27` and overriding my changes.

Comment (paulhcsun) [reply packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json]: Realized I was not updating the value in the correct place. It should be fixed now.

**Comment by paulhcsun** [comment]
oops one more integ test snapshot to update

**Comment by paulhcsun** [comment]
@mergifyio update

**Comment by paulhcsun** [comment]
@Mergifyio update

**Pull request merged**